meta-mender-core/recipes-mender/mender-client/mender-client-cpp.inc

```diff
@@ -97,7 +97,7 @@ FILES:mender-auth += "\
     ${systemd_unitdir}/system/mender-authd.service \
 "
 
-DEPENDS += "lmdb googletest boost libarchive"
+DEPENDS += "lmdb googletest boost libarchive dbus"
 RDEPENDS:mender-auth:append = " boost-filesystem"
 RDEPENDS:mender-update:append = " boost-filesystem libarchive lmdb"
```

Comment:
And runtime dependency..? Probably it doesn't matter that much since `dbus` is required for all modern Linuxes, but for consistency.

Reply:
Good point!